exaudfclient/base/exaudflib/impl/swig/swig_meta_data.cc

```diff
@@ -0,0 +1,2 @@
+#include "exaudflib/impl/swig/swig_meta_data.h"
+
```

Comment:
Explain why we need this almost empty file

Reply:
This is a common way to verify that the header files contains all necessary includes.